website/data-storage/src/consumer/consumer.controller.ts

```diff
@@ -14,15 +14,15 @@ import {
   KafkaPayload,
   Keys,
   NewEventData,
+  UpdateMessageParams,
 } from '@gear-js/common';
 
 import { ConsumerService } from './consumer.service';
 
-const logger = new Logger('ConsumerController');
-
 @Controller()
 export class ConsumerController {
-  constructor(private readonly consumerService: ConsumerService) {}
+  private logger: Logger = new Logger('ConsumerController');
```

Comment:
@osipov-mit  In my opinion, it will be better to use no in class if the `logger` is specified with a special `title` like `ConsumerController`. I don't see any benefits to making out of scope `ConsumerController` class. We have known that the `ConsumerController` `logger` only has used in the `class`.